tasks/task1.js

```diff
@@ -0,0 +1,92 @@
+/* Task 1 */
+function summ(...args) {
```

Comment:
Add check for existence (if undefined -> return) to avoid errors

(*for every function)

Reply:
check that value exists, not only for undefined